diff -up taglib-1.12/taglib-config.cmake.1~ taglib-1.12/taglib-config.cmake
--- taglib-1.12/taglib-config.cmake.1~	2022-01-08 23:49:15.475181794 +0100
+++ taglib-1.12/taglib-config.cmake	2022-01-08 23:49:48.834292115 +0100
@@ -16,7 +16,6 @@ EOH
 
 prefix=@CMAKE_INSTALL_PREFIX@
 exec_prefix=@CMAKE_INSTALL_PREFIX@
-libdir=@LIB_INSTALL_DIR@
 includedir=@INCLUDE_INSTALL_DIR@
 
 flags=""
@@ -29,10 +28,10 @@ while test $# -gt 0
 do
   case $1 in
     --libs)
-	  flags="$flags -L$libdir -ltag @ZLIB_LIBRARIES_FLAGS@"
+	  flags="$flags -ltag @ZLIB_LIBRARIES_FLAGS@"
 	  ;;
     --cflags)
-	  flags="$flags -I$includedir -I$includedir/taglib"
+	  flags="$flags -I$includedir/taglib"
 	  ;;
     --version)
 	  echo @TAGLIB_LIB_VERSION_STRING@
